sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/JobsService.java

```diff
@@ -110,13 +113,24 @@ public JobView createJobView(JobView jobView) {
         return jobView;
     }
 
+    @Transactional(rollbackFor = Exception.class)
```

Comment:
Please remove `(rollbackFor = Exception.class)` wherever you perform refactoring. It works for RuntimeException by default and that's more than enough here.